plugins/woocommerce-admin/client/core-profiler/actions/tracks.tsx

```diff
@@ -154,11 +154,20 @@ const recordFailedPluginInstallations = (
 	_context: unknown,
 	_event: PluginsInstallationCompletedWithErrorsEvent
 ) => {
+	const failedExtensions = _event.payload.errors.map(
+		( error: PluginInstallError ) => getPluginTrackKey( error.plugin )
+	);
+
 	recordEvent( 'coreprofiler_store_extensions_installed_and_activated', {
 		success: false,
-		failed_extensions: _event.payload.errors.map(
-			( error: PluginInstallError ) => getPluginTrackKey( error.plugin )
-		),
+		failed_extensions: failedExtensions,
+	} );
+
+	failedExtensions.forEach( ( extension ) => {
+		recordEvent( 'coreprofiler_store_extension_installed_and_activated', {
+			success: false,
+			extension,
```

Comment:
Should we also include `error_message` here?

Reply:
Yep 👍  included it in 8ea18529bd9278a2151b35591e6e6907b8a2e0ca.

![Screenshot 2024-01-09 at 11 37 27](https://github.com/woocommerce/woocommerce/assets/4344253/598068ac-ce1b-4be2-abf3-0c82c2a808c8)

Reply:
![Screenshot 2024-01-09 at 11 38 32](https://github.com/woocommerce/woocommerce/assets/4344253/a73e2d20-098e-40a6-a84e-209c92093670)

Reply:
Awesome, thanks!